package.json

```diff
@@ -10,13 +10,13 @@
     "test": "nyc mocha"
   },
   "dependencies": {
-    "@octokit/core": "2.1.2",
-    "@octokit/plugin-throttling": "2.7.0",
+    "@octokit/core": "^2.2.0",
```

Comment:
I added the `^` here. It was only missing because of my habit of using `npm install --save-exact` for other repos. It should be all or nothing here IMHO.